python/taichi/main.py

```diff
@@ -88,9 +88,14 @@ def make_argument_parser():
 
 def main(debug=False):
     argc = len(sys.argv)
-    mode = 'help' if argc == 1 else sys.argv.pop(1)
```

Comment:
`sys.argv.pop` modifies `sys.argv`, which makes a few commands not functioning well.

Reply:
Nice catch!